src/index.js

```diff
@@ -96,9 +96,14 @@ var OAuth = function OAuth(clientId) {
 	this.refreshToken = null;
 	this.tokenType = null;
 	this.expiresIn = 0;
-	this.ignoreSslError = false;
 };
 
+
+/**
+ * For by passing self signed certificate. By default false
+ */
+OAuth.prototype.ignoreSslError = false;
```

Comment:
If it is not assigned it that way then how can client set value of ignoreSslError?